plenum/test/monitoring/test_EMALatencyMeasurementForAllClient.py

```diff
@@ -0,0 +1,77 @@
+import copy
+
+import pytest
+
+from plenum.server.monitor import EMALatencyMeasurementForAllClient, EMALatencyMeasurementForEachClient
```

Comment:
The file name is `test_EMALatencyMeasurementForAllClient.py` while we are testing `EMALatencyMeasurementForEachClient`.
Can we test both BTW?